From 24323f62a61516add66076620c76d41866d0855d Mon Sep 17 00:00:00 2001
From: Davide Depau <davide@depau.eu>
Date: Sun, 10 Jan 2021 02:59:27 +0100
Subject: [PATCH] arm64: dts: rockchip: Orange Pi 4 SDHCI/SDIO: allow lower
 rates

The system sometimes doesn't boot due to timeouts in HS400es setup.
This also allows HS200, or at least I hope so.

Signed-off-by: Davide Depau <davide@depau.eu>
---
 arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
index ad72a5ba527f..2ad00db1ed87 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
@@ -892,11 +892,12 @@ &saradc {
 
 &sdhci {
 	bus-width = <8>;
+	mmc-hs200-1_8v;
 	mmc-hs400-1_8v;
-	supports-emmc;
-	non-removable;
-	keep-power-in-suspend;
 	mmc-hs400-enhanced-strobe;
+	//supports-emmc;
+	non-removable;
+	//keep-power-in-suspend;
 	status = "okay";
 };
 
-- 
2.30.0

